pages/running_tests.md

```diff
@@ -59,7 +59,7 @@ UI unit tests are written in [Jasmine](http://jasmine.github.io) and run with [K
 
 Those tests will mock up the access to the application's REST endpoints, so you can test your UI layer without having to launch the Java back-end.
 
-*   Those tests can be run using `gulp test`.
+*   Those tests can be run using `yarn test` (For NPM: `gulp test`).
```

Comment:
Can you change "For NPM" to "For AngularJS", because `gulp` only exists for AngularJS projects.

`npm test` will work the same as `yarn test` in new Angular projects